draft-ietf-taps-impl.md

```diff
@@ -722,8 +722,13 @@ This enables implementations to realize transparent connection coalescing, conne
+When a path change occurs, e.g. when the IP address of an interface changes, this will usually break connectivity on that path for any active connection. Still, the Transport Services implementation is responsible for notifying Protocol Instances in the Protocol Stack if path changes are detected in order to quickly react to path failures before a time-out occurs.
+
+For protocols that do not support multipath or migration, the Protocol Instances should be informed of the path change, but should not be forcibly disconnected if the previously used path becomes unavailable. An exception to this case is if the System Policy changes to prohibit traffic from the Connection based on its properties, in which case the Protocol Stack should be disconnected.
+
+If the Protocol Stack includes a transport protocol that supports multipath connectivity also in form of migration support, the Transport Servics implementation should also inform the Protocol Instance of potentially new paths that are permissible based on the Selection Properties passed by the application. A protocol can establish new subflows over new paths as long as an active path is still available or, if migration is supported, also after the break has been detected, and should attempt tear down subflows over paths that are no longer used. The taps API provides an interface to set a multipath policy that indicates when and how different paths should be used, however, detailed handling of these policies is still implementation-specific. Further the decision when to create a new path or announce a new path or set of paths to the other end, e.g. in form of additional IP addresses, is implementation specific or maybe be supported by future API extensions. If the Protocol Stack includes a transport protocol that does not support multipath, but support migrating between paths, the update to available paths can be used as the trigger for migrating the connection. 
+
+Pooled Connections {{pooled-connections}} may add or remove underlying transport connections in a similar manner. 
```

Comment:
```suggestion
In case of Pooled Connections {{pooled-connections}}, the transport system may add connections over new paths or different protocols to the pool if permissible based on the multipath policy and Selection Properties. In case a previously used path becomes unavailable, the transport system may disconnect all connections that require this path, but should not disconnect the pooled connection object exposed to the application. The strategy how is implementation specific, but should me consistent with the behaviour of multipath transports. 
```